deploy/configure_tls_trust.sh

```
@@ -0,0 +1,57 @@
+    2)
+      echo "Using enable_insecure for development/self-signed environments."
+      export TRUST_HELM_ARGS='--set enable_insecure=true'
+      ;;
+    3)
+      read -r -p "Name of the Secret containing PEM-encoded CA cert(s) [company-internal-ca]: " ca_secret
+      ca_secret=${ca_secret:-company-internal-ca}
+      ca_secret_key="ca.crt"
+
+      if kubectl -n "$ns" get secret "$ca_secret" >/dev/null 2>&1; then
+        if ! kubectl -n "$ns" get secret "$ca_secret" -o "jsonpath={.data.${ca_secret_key}}" | grep -q .; then
+          available_keys=$(kubectl -n "$ns" get secret "$ca_secret" -o go-template='{{range $k,$v := .data}}{{printf "%s " $k}}{{end}}')
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)